client/src/App.css

```diff
@@ -26,3 +26,7 @@
   from { transform: rotate(0deg); }
   to { transform: rotate(360deg); }
 }
+
+#map {
```

Comment:
Do we need an ID as a css selector? I'm wondering if you could just use class.

Reply:
ID or class, I think they both work. I can change it to class.

Reply:
When I change it to className, I get this error, but it still works:
`js?key=AIzaSyBA2RlpJOlVVkN9u7HfZRcKEgfG2Nxc35U&libraries=places&callback=googleMapsLoaded:87 Uncaught TypeError: Cannot read property 'firstChild' of null
    at Object._.Dg (js?key=AIzaSyBA2RlpJOlVVkN9u7HfZRcKEgfG2Nxc35U&libraries=places&callback=googleMapsLoaded:87)
    at new Hg (js?key=AIzaSyBA2RlpJOlVVkN9u7HfZRcKEgfG2Nxc35U&libraries=places&callback=googleMapsLoaded:88)
    at initMap (googleMaps.js:23)
    at ./src/features/app/googleMaps.js.window.googleMapsLoaded (googleMaps.js:12)
    at js?key=AIzaSyBA2RlpJOlVVkN9u7HfZRcKEgfG2Nxc35U&libraries=places&callback=googleMapsLoaded:96
    at js?key=AIzaSyBA2RlpJOlVVkN9u7HfZRcKEgfG2Nxc35U&libraries=places&callback=googleMapsLoaded:65
    at js?key=AIzaSyBA2RlpJOlVVkN9u7HfZRcKEgfG2Nxc35U&libraries=places&callback=googleMapsLoaded:63
    at Array.<anonymous> (js?key=AIzaSyBA2RlpJOlVVkN9u7HfZRcKEgfG2Nxc35U&libraries=places&callback=googleMapsLoaded:65)
    at js?key=AIzaSyBA2RlpJOlVVkN9u7HfZRcKEgfG2Nxc35U&libraries=places&callback=googleMapsLoaded:123
    at js?key=AIzaSyBA2RlpJOlVVkN9u7HfZRcKEgfG2Nxc35U&libraries=places&callback=googleMapsLoaded:63
    at js?key=AIzaSyBA2RlpJOlVVkN9u7HfZRcKEgfG2Nxc35U&libraries=places&callback=googleMapsLoaded:123
    at Zd (js?key=AIzaSyBA2RlpJOlVVkN9u7HfZRcKEgfG2Nxc35U&libraries=places&callback=googleMapsLoaded:65)
    at Yd.ua (js?key=AIzaSyBA2RlpJOlVVkN9u7HfZRcKEgfG2Nxc35U&libraries=places&callback=googleMapsLoaded:123)
    at js?key=AIzaSyBA2RlpJOlVVkN9u7HfZRcKEgfG2Nxc35U&libraries=places&callback=googleMapsLoaded:149`
I think I'll change it back to ID for less errors.

Reply:
I also just remembered we are using style components! This would be a great chance to test that out. The errors you got were because of the js selector to select element by ID, and not caused by CSS.
The best thing to do would be to add a className to this, and then style it with styled components!

Reply:
OK found out that the map is actually an array and in the first position.
This finally got the map to show:
document.getElementsByClassName('map')[0]. This commit is now pushed.